client/selectors/events.ts

```diff
@@ -47,7 +52,7 @@ export const orderedEvents = createSelector(
             }];
         }
 
-        const dateRange = getSearchDateRange(search, appConfig.start_of_week);
+        const dateRange = getSearchDateRange(search, appConfig.start_of_week, viewInterval);
```

Comment:
Will this be required for [selectors/planning.ts](https://github.com/superdesk/superdesk-planning/blob/develop/client/selectors/planning.ts#L85) as well, so the Planning view is capped to the same date range logic?

Reply:
yep makes sense to be consistent